src/lazyfree.c

```diff
@@ -215,9 +215,7 @@ void freeFunctionsAsync(functionsLibCtx *functions_lib_ctx) {
 
 /* Free replication backlog referencing buffer blocks and rax index. */
 void freeReplicationBacklogRefMemAsync(list *blocks, rax *index) {
-    if (listLength(blocks) > LAZYFREE_THRESHOLD ||
-        raxSize(index) > LAZYFREE_THRESHOLD)
```

Comment:
this isn't necessarily wrong (or right).
O(n) is the same complexity as O(2n).

for example freeFunctionsAsync counts the number of functions, but functionsLibCtxFree releases several data structures (quite a lot of them actually).

it it can go either way, then i think that even if i like your version better, i'll opt out to keep the code unchanged instead.

Reply:
ok i am closing it.
my first thought was to make them consistent, since it is a trivial change